types/geodesy/geodesy-tests.ts

```diff
@@ -54,23 +54,23 @@ Dms.parse('51° 28′ 40.12″ N');
 
 Dms.toDms(45);
 Dms.toDms(45, 'dm');
-Dms.toDms(45, 'd', 2);
-Dms.toDms(45, 'dms', 4);
+Dms.toDms(45, 'd', 1);
+Dms.toDms(45, 'dms', 3);
```

Comment:
The intend of the changed tests is that one can define a number of decimal points that are different from the default values.